localbuild/meta.yaml

```diff
@@ -66,7 +66,7 @@ requirements:
     - flask-mail
     - flask-migrate
     - werkzeug >=2.2.3
-    - flask-socketio =5.1.0
+    - flask-socketio
```

Comment:
may be better >=5.1.0

the recent version is 5.3.5

You could try to make a local build, boa is currently the fastest method.

e.g.
```
boa convert meta.yaml > recipe.yaml
boa build .
```

```

Reply:
Done. A local boa build looks good as well.